### ISSUE TITLE
Repo - About

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#1951]
User interfaces can be found [here](https://zpl.io/DplNkR3).

In V3, the proposed area of the template to house 'About' information is in the structured lists in the Overview tab. This is intended to group this content with similar information as specified in the FHIR specs, to visually and mentally start aligning our users towards a more consistent presentation and organisation of how they describe their content.

Please record any feedback that needs to be actioned before moving the Dev here.

For future requirements, let's capture in a separate ticket and prioritise accordingly.

**Comment by jamlung-ri** [comment]
@paulsonder My only comment is about displaying the Repo's Long name, since a repo can have both a long or short name. 

Once that comment is addressed, then this design is ready to move into Dev!

**Comment by paulsonder** [comment]
Thanks @jamlung-ri - the full name has also been added to the design.

**Comment by paulsonder** [comment]
@snyaggarwal Moving to **'Ready for dev'** - let me know if you would like to schedule some time to discuss.

**Comment by snyaggarwal** [comment]
@jamlung-ri Can you please help list down fields that should go in each section of About page. The design doesn't have all fields.

**Comment by snyaggarwal** [comment]
@jamlung-ri @paulsonder do we have an empty state for this? because everything here is optional content.

**Comment by paulsonder** [comment]
Hi @snyaggarwal - I've added an Empty state to [Zeplin](https://zpl.io/YPYD8eM).

**Comment by snyaggarwal** [comment]
@paulsonder fields that are missing from Spreadsheet (don't have write access):
1. Created On (created_at)
2. Created By (created_by)
3. Custom Attributes (extras)
4. Locales (default_locale and supported_locales)
5. Website (website)
6. External ID (external_id)
7. Is Processing (is_processing) -- used to be warning color chip in v2
8. content_type (FHIR)
9. revision_date (FHIR)
10. case_sensitive (FHIR)
11. collection_reference (FHIR)
12. compositional (FHIR)
13. meta (FHIR)
14. Hierarchy Meaning (hierarchy_meaning)
15. Hierarchy Root (hierarchy_root)
16. Auto ID Concept ID
17. Auto ID Concept External ID
18. Auto ID Mapping ID
19. Auto ID Mapping External ID
20. Auto ID Concept Name ID
21. Auto ID Concept Description ID
22. Auto ID Concept ID Start From
23. Auto ID Concept External ID Start From
24. Auto ID Mapping ID Start From
25. Auto ID Mapping External ID Start From

**Comment by paulsonder** [comment]
Thank you @snyaggarwal, really useful to have this collected.

I'll add to the Sheet and check we've got a place in the design for the content.
